packages/evm/README.md

```diff
@@ -25,22 +25,19 @@ Note that this package atm cannot be run in a standalone mode but needs to be ex
 
 ```typescript
 import { Chain, Common, Hardfork } from '@ethereumjs/common'
-import { Blockchain } from '@ethereumjs/blockchain'
 import { EEI } from '@ethereumjs/vm'
 import { EVM } from '@ethereumjs/evm'
 import { DefaultStateManager } from '@ethereumjs/statemanager'
 
 // Note: in a future release there will be an EEI default implementation
 // which will ease standalone initialization
 const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.London })
-const blockchain = await Blockchain.create({ common })
 const stateManager = new DefaultStateManager({ common })
 const eei = new EEI(stateManager, common, blockchain)
```

Comment:
This has still (minimally) two things in it which breaks the example:

1. the `DefaultStateManager()` doesn't accept a `common` object any more
2. `blockchain` is still passed to EEI but initialization has now been removed by the PR

Sorry guys, but: 1 Submitter, 1 Reviewer, this is a bit too much for a 3 LOC change. 🤨 Can you please *run* an example you want to improve at least once?